unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/Scripts/GodMode/Inspector/EntityListAdapter.cs

```diff
@@ -9,8 +9,8 @@ public class EntityListAdapter : MonoBehaviour
 {
```

Comment:
Putting this here because it's not part of the PR, but I feel compelled to comment about it.

Here:
https://github.com/decentraland/unity-renderer/blob/caa54a5f2476eee91dec0fdad0f2f6e5069f55e5/unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/Scripts/GodMode/Inspector/EntityListAdapter.cs#L171

Can we please use braces or && operator for the nested if statements? If we have a conflict here it will be harder to solve than it should.